perllib/FixMyStreet/App/Controller/Offline.pm

```diff
@@ -35,12 +35,27 @@ sub fallback : Local {
     my ($self, $c) = @_;
 }
 
-sub manifest: Path("/.well-known/manifest.webmanifest") : Args(1) {
-    my ($self, $c, $site) = @_;
+sub manifest_waste: Path('/.well-known/manifest-waste.webmanifest') {
+    my ($self, $c) = @_;
+
+    $c->stash->{manifest_application} = 'ww';
```

Comment:
This works fine, but rather than adding more data to the stash, you could pass the data through directly, ie. `$c->forward('send_manifest', [ 'ww' ]);`, and then send_manifest first line would be `my ($self, $c, $app) = @_;`